great_expectations/expectations/expectation.py

```diff
@@ -1238,6 +1239,19 @@ def configuration(self) -> ExpectationConfiguration:
     def __copy__(self):
         return self.copy(update={"id": None})
 
+    def copy(self, deep: bool = True) -> Self:
+        new_expectation = self.__class__(
+            id=None,
+            meta=self.meta,
+            notes=self.notes,
+            result_format=copy.deepcopy(self.result_format),
+            catch_exceptions=self.catch_exceptions,
+            rendered_content=copy.deepcopy(self.rendered_content),
+            **self.configuration.kwargs,
```

Comment:
It just occurred to me the Expectation is a pydantic model and there might be non-kwarg additional arguments. Maybe we can leverage `__fields_set__` or some other pydantic metadata to set the arguments correctly. I can followup on Monday.

Reply:
pydantic models don't accept positional arguments unless we have defined a custom `__init__`.

Do you mean we need to deal with `extra` kwargs that don't have a defined field?
If so, then yes, I think `__fields_set__` would include any extra fields that have been provided.